app/src/main/res/layout/fragment_settings.xml

```diff
@@ -1,65 +1,61 @@
 <?xml version="1.0" encoding="utf-8"?>
-<layout>
```

Comment:
Disabling databinding here creates a bug with settings switch not saving preferencekey data correctly, which is why I keep this with databinding, refer to #272 for more info. Considering you removed all binding adapters maybe we should set keys programmatically then, instead of using xml and BindingAdapters